src/tools/rbd/ArgumentTypes.cc

```diff
@@ -282,7 +282,8 @@ void add_create_image_options(po::options_description *opt,
     (IMAGE_SHARED.c_str(), po::bool_switch(), "shared image")
     (IMAGE_STRIPE_UNIT.c_str(), po::value<ImageObjectSize>(), "stripe unit in B/K/M")
     (IMAGE_STRIPE_COUNT.c_str(), po::value<uint64_t>(), "stripe count")
-    (IMAGE_DATA_POOL.c_str(), po::value<std::string>(), "data pool");
+    (IMAGE_DATA_POOL.c_str(), po::value<std::string>(), "data pool")
+    (IMAGE_EAGER_THICK.c_str(), po::bool_switch(), "eager-thick image provisioned");
```

Comment:
This applies to ```import```, ```clone```, and ```copy``` commands as well as ```create```.  Since it really doesn't make sense for those other cases, you should only add the argument to the ```create``` argument set within ```Create.cc```